ci/scripts/check-ascii.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
```

Comment:
Is it bad that I think this is a bit too complicated and maybe should just be a dumb python script

Reply:
Well, this directory is full of simple bash scripts. If you want to rewrite them all in Python, I guess you can... :-)

Reply:
Python is certainly nicer for the regex-preparing string manipulations in this script, but less so for the simplicity of the pipeline which performs the check (`git ls-files | grep filenames | xargs grep for non ascii`).

I'd suggest that the string-concatenation bit of this script should be put into a function, but even then, after all the bits are concatenated together, there is some final preparation of the regex which is slightly different between the two.